react-native-scripts/template/app.json

```diff
@@ -1,5 +1,5 @@
 {
   "expo": {
-    "sdkVersion": "15.0.0"
+    "sdkVersion": "UNVERSIONED"
```

Comment:
of course we will change this before merging, just changed it for now so you can test